src/components/Logo/Logo.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react'
 
-const Logo = ({ abbreviated = false }: { abbreviated?: boolean }) => {
+const Logo = ({
+  abbreviated = false,
+  darkBg = false,
+}: {
+  abbreviated?: boolean
+  darkBg?: boolean
+}) => {
+  if (darkBg) {
+    return (
+      <img
+        src="/assets/images/SpaceForce_Horizontal_Gradient_RGBmod-01.svg"
+        alt="United States Space Force"
```

Comment:
looks like we need to update the unit tests w the alt text

Reply:
thanks!